webapp/src/components/Metrics/PerformanceAnalysisTable.tsx

```diff
@@ -253,6 +262,7 @@ const PerformanceAnalysisTable: React.FC<Props> = ({
                         display="flex"
                         alignItems="center"
                         justifyContent="center"
+                        zIndex="tooltip"
```

Comment:
That is not a tooltip. I'm afraid putting it at the `zIndex` of tooltips might cause conflicts. A simple `1` seems to work. Makes sense?
```suggestion
                        zIndex={1}
```

Reply:
My initial choice was to use zIndex={1} but chose to add zIndex="tooltip" because this is specifically for a tooltip.

Reply:
No, a [tooltip](https://en.wikipedia.org/wiki/Tooltip) only appears
> when hovering over a screen element or component